src/nbl/ext/DebugDraw/CDrawAABB.cpp

```diff
@@ -73,96 +75,75 @@ core::smart_refctd_ptr<DrawAABB> DrawAABB::create(SCreationParameters&& params)
     return core::smart_refctd_ptr<DrawAABB>(new DrawAABB(std::move(constructorParams)));
 }
 
-// note we use archive entry explicitly for temporary compiler include search path & asset cwd to use keys directly
-constexpr std::string_view NBL_ARCHIVE_ENTRY = _ARCHIVE_ENTRY_KEY_;
+// extension data mount alias
+constexpr std::string_view NBL_EXT_MOUNT_ENTRY = "nbl/ext/DebugDraw";
 
-const smart_refctd_ptr<IFileArchive> DrawAABB::mount(smart_refctd_ptr<ILogger> logger, ISystem* system, const std::string_view archiveAlias)
+const smart_refctd_ptr<IFileArchive> DrawAABB::mount(smart_refctd_ptr<ILogger> logger, ISystem* system, video::ILogicalDevice* device, const std::string_view archiveAlias)
```

Comment:
this function deals with archives and file system, why is `ILogicalDevice` taken?

Reply:
https://github.com/Devsh-Graphics-Programming/Nabla/pull/900/commits/1badc7a3a911c88e48227c017af65a1c46e6db39